src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js

```diff
@@ -46,13 +46,13 @@ const MobileWorkingWorkOrdersView = ({ currentUser }) => {
     setIsLoading(true)
 
     try {
-      const url = featureToggleStatus?.fetchAppointmentsFromDrs
-        ? `/api/operatives/${currentUser.operativePayrollNumber}/appointments`
-        : `/api/operatives/${currentUser.operativePayrollNumber}/workorders`
+      // const url = featureToggleStatus?.fetchAppointmentsFromDrs
```

Comment:
Should remove this commented code - we can get it back from git history if needed